contracts/src/v0.8/dev/automation/2_1/KeeperRegistry2_1.sol

```diff
@@ -217,6 +217,7 @@ contract KeeperRegistry2_1 is KeeperRegistryBase2_1, OCR2Abstract, Chainable, ER
     s_storage.nonce++;
     s_upkeepOffchainConfig[id] = offchainConfig;
     emit UpkeepRegistered(id, gasLimit, admin);
+    emit UpkeepOffchainConfigSet(id, offchainConfig);
```

Comment:
can you run `go generate ./core/gethwrappers/... ` to update the go wrapper for 2.1?

Reply:
There are no file changes after running go gen